pkg/nss/interface.go

```diff
@@ -0,0 +1,23 @@
+package nss
+
+// Logger is the interface used by the NSS package to log messages.
+type Logger interface {
+	Debugf(format string, v ...interface{})
+	Errorf(format string, v ...interface{})
+}
+
+// CertUtilServicer encapsulates operations on NSS database using certutil command.
+type CertUtilServicer interface {
```

Comment:
Non blocking,

Is it a common thing to name interfaces that way ? 
IMO, `CertUtilServiceInterface` would be clearer since we directly know that it is an interface

Reply:
> sometimes to refer to CertUtilService and other times to CertUtilServicer. Is it the same think (hence a typo)?

@Thykof `CertUtilServicer` is the interface, and `CertUtilService` a struct implementing the interface

WDYT about this suggestion ?

Reply:
Ho I see, thanks.

IMO no we should not name a interface BlaBlaInterface, not IBlaBla for many reason I found on internet.

Reply:
The interface are named XXXer mostly when they contain only one function:
```golang
type Writer interface {
     Write(file string, content []byte) error
}
```
Here servicer is not an english word